spec/models/audits/audit_spec.rb

```diff
@@ -1,4 +1,4 @@
-RSpec.describe Audit do
+RSpec.describe Audits::Audit do
```

Comment:
If you enclose the test within `module Audits` there is no need to scope every class inside